packages/flutter/lib/src/rendering/flex.dart

```diff
@@ -588,6 +591,22 @@ class RenderFlex extends RenderBox with ContainerRenderObjectMixin<RenderBox, Fl
     }
   }
 
+  /// How much space to place between children in a run in the main axis.
+  ///
+  /// For example, if [spacing] is 10.0, the children will be spaced at least
+  /// 10.0 logical pixels apart in the main axis.
+  ///
+  /// Defaults to 0.0.
```

Comment:
this should mention explicitly if the padding is only applied between children or also around children, and how it interacts with the main axis alignments that spread out the children, especially in the case of there not being enough space (e.g. will the spacing be ignored in favour of fitting the children, or will the children just overflow).

Reply:
Updated documentation for different main axis alignemtns and overflow when total children sizes and spacing can overflow if the layout is over constrained.